epub2speech/convertor.py

```
@@ -113,6 +114,37 @@ def _convert_chapter_to_audio(
+                elif format_name == 'WEBP':
+                    extension = '.webp'
+                else:
+                    extension = '.jpg'
+
+                cover_path = self._workspace_path / f"cover{extension}"
+
+                # 保存图片
+                img.save(cover_path, format=format_name)
+                return cover_path
+
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)